### PR TITLE
Update Booking.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

**Comment by cschanaj** [comment]
Note that this is still `default_off`

**Comment by koops76** [comment]
@cschanaj Fixed.

File: src/chrome/content/rules/Booking.com.xml [hunk 1]
```diff
@@ -14,15 +8,19 @@ Fetch error: http://s-ec.bstatic.com/ => https://s-ec.bstatic.com/: Too many red
 	Nonfunctional domains:
 
 		- blog.booking.com ¹
+		- r-ec.bstatic.com ʰ
 		- suite.booking.com ʳ
+		- s-ec.bstatic.com ʰ
```
Comment (J0WI) [inline src/chrome/content/rules/Booking.com.xml]: The rule will still redirect them.

**Comment by J0WI** [comment]
See also #7558

File: src/chrome/content/rules/Booking.com.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Booking.com.xml]: @J0WI Fixed.

**Comment by J0WI** [comment]
Could you also merge the changes from #7558?

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/Booking.com.xml [hunk 2]
```diff
@@ -5,16 +5,18 @@
...
-		- blog.booking.com ¹
-		- r-ec.bstatic.com ʰ
-		- suite.booking.com ʳ
-		- s-ec.bstatic.com ʰ
+		- booking.com
+ 			- blog		¹
+ 			- cs		²
+ 			- m.admin	(cert mismatched, CN: admin.booking.com)
+ 			- office	²
+ 			- suite		ʳ
```
Comment (J0WI) [inline src/chrome/content/rules/Booking.com.xml]: Please readd the comments for `(r|s)-ec`

File: src/chrome/content/rules/Booking.com.xml [hunk 3]
```diff
@@ -5,16 +5,18 @@
...
 
-	Nonfunctional domains:
+	Nonfunctional subdomains:
 
-		- blog.booking.com ¹
-		- r-ec.bstatic.com ʰ
-		- suite.booking.com ʳ
-		- s-ec.bstatic.com ʰ
+		- booking.com
+ 			- blog		¹
```
Comment (J0WI) [inline src/chrome/content/rules/Booking.com.xml]: WFM

File: src/chrome/content/rules/Booking.com.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/Booking.com.xml]: @J0WI Done.

File: src/chrome/content/rules/Booking.com.xml (hunk 3 above)
Comment (koops76) [inline src/chrome/content/rules/Booking.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Please have a look at the failing tests.

**Comment by koops76** [comment]
@J0WI Fixed.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**